modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -176,40 +188,77 @@ class SegmentEdit extends Component {
 		);
 	};
 
+	_renderLocalizedInputs = () => {
+		const {
+			defaultLanguageId,
+			portletNamespace,
+			values
+		} = this.props;
+
+		const langs = Object.entries(values.name);
```

Comment:
`Object.entries` isn't supported in IE11, we should probably handle this another way.

https://caniuse.com/#feat=object-entries

Reply:
Hi Bryce, `Object.entries` is fully polyfilled in the portal and is used quite a lot in the WEM editor. However I just made the switch to `Object.keys`. Thanks!!

Reply:
Ah I see, I wasn't aware that we had a polyfill for it. Thanks for letting me know!